jenkinsapi/jenkins.py

```diff
@@ -22,13 +22,16 @@
 from jenkinsapi.nodes import Nodes
```

Comment:
Please revert this change. This is already done by parent class.

Reply:
I'll be happy to remove it, but I can't recall having added the line!

Reply:
I mean whole change made in commit 50f17a0 to `jenkins.py` is not needed and must be reverted.

Reply:
is there a better way to request the current state of api/python?

Reply:
do you still want me to remove: "from jenkinsapi.nodes import Nodes"
This i not a added by me?